service/src/service.rs

```diff
@@ -110,6 +110,9 @@ pub struct PostService {
     threads: post::config::Cores,
     pow_flags: RandomXFlag,
     proof_generation: Mutex<ProofGenProcess>,
+    remote_k2pow: Option<String>,
+    remote_k2pow_parallelism: usize,
+    remote_k2pow_backoff: u64,
```

Comment:
How about enclosing it in a Config struct?

```rust
pub struct K2powConfig {
    url: String,
    parallelilsm: usize
    backoff: time::Duration,
}
```

```suggestion
    remote_k2pow: Option<K2powConfig>,
```